assets/src/dashboard/app/views/shared/telemetryBanner.js

```diff
@@ -153,6 +175,10 @@ export const TelemetryOptInBanner = forwardRef(
               href={__('https://policies.google.com/privacy', 'web-stories')}
```

Comment:
```suggestion
              href={'https://policies.google.com/privacy'}
```

don't think we want to translate this

Reply:
We do want that :-)

The privacy policy is available in different languages.

For Spanish, translators can translate this string to `https://policies.google.com/privacy?hl=es`

Reply:
ah cool, today i learned :)